tests/plugins/test_custom_plugin.py

```diff
@@ -0,0 +1,22 @@
+from pytorch_lightning import Trainer
+from pytorch_lightning.plugins import DDPPlugin
+from tests.helpers import BoringModel
+
+
+class CustomParallelPlugin(DDPPlugin):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
This would work too, right?

```suggestion
class CustomParallelPlugin(DDPPlugin): ...
```